src/app/components/application/review-registration/parent-review/parent-review.component.html

```diff
@@ -34,6 +34,36 @@ <h4>Parent or Guardian: {{parent.firstName}} {{parent.lastName}}</h4>
       <div class="col-7 hideOverflow" *ngIf="parent.email">{{parent.email}}</div>
     </div>
 
+    <div class="review my-1 row">
+      <div class="col-5 hideOverflow">Preferred Phone</div>
+      <div class="col-7 hideOverflow" *ngIf="!parent.preferredPhone">
+        <i class="fa fa-warning" style="color:red"></i><span class="reviewError"> Not Provided</span>
+      </div>
+      <div class="col-7 hideOverflow" *ngIf="parent.preferredPhone">{{parent.preferredPhone}}</div>
+    </div>
+
+    <div class="review my-1 row" *ngIf="parent.preferredPhone && parent.preferredPhone === 'Home'">
+      <div class="col-5 hideOverflow">Home Phone</div>
+      <div class="col-7 hideOverflow" *ngIf="!parent.homePhone">
+        <i class="fa fa-warning" style="color:red"></i><span class="reviewError"> Not Provided</span>
+      </div>
+      <div class="col-7 hideOverflow" *ngIf="parent.homePhone">{{parent.homePhone}}</div>
+    </div>
+
+    <div class="review my-1 row" *ngIf="(parent.preferredPhone && parent.preferredPhone === 'Mobile') || parent.optIn">
+      <div class="col-5 hideOverflow">Mobile Phone</div>
+      <div class="col-7 hideOverflow" *ngIf="!parent.mobilePhone">
+        <i class="fa fa-warning" style="color:red"></i><span class="reviewError"> Not Provided</span>
+      </div>
+      <div class="col-7 hideOverflow" *ngIf="parent.mobilePhone">{{parent.mobilePhone}}</div>
+    </div>
```

Comment:
Shouldn't the overall display of these be based on if we have a value or not? I'd expect that even if it wasn't "preferred" we'd show them on the review page that we have a value for each phone they gave us.

Reply:
Yep, my bad. I missed that